CHANGELOG.md

```diff
@@ -1,5 +1,90 @@
 # Database Tools Changelog
 
+## 100.4.0
+
+_Released 2021-07-19_
+
+We are pleased to announce version 100.4.0 of the MongoDB Database Tools.
+
+This release includes server 5.0 support, including dump/restoring timeseries collections.  
+
+The Database Tools are available on the [MongoDB Download Center](https://www.mongodb.com/try/download/database-tools).
+Installation instructions and documentation can be found on [docs.mongodb.com/database-tools](https://docs.mongodb.com/database-tools/).
+Questions and inquiries can be asked on the [MongoDB Developer Community Forum](https://developer.mongodb.com/community/forums/tags/c/developer-tools/49/database-tools).
+Please make sure to tag forum posts with `database-tools`.
+Bugs and feature requests can be reported in the [Database Tools Jira](https://jira.mongodb.org/browse/TOOLS) where a list of current issues can be found.
+
+<h3>        Build Failure
+</h3>
+<ul>
+<li>[<a href='https://jira.mongodb.org/browse/TOOLS-2892'>TOOLS-2892</a>] -         aws-auth tests failing on all variants
+</li>
+<li>[<a href='https://jira.mongodb.org/browse/TOOLS-2893'>TOOLS-2893</a>] -         legacy-js-tests 4.4 and 5.0 failing on all variants
+</li>
+</ul>
+                                                                        
+<h3>        Release
+</h3>
+<ul>
+<li>[<a href='https://jira.mongodb.org/browse/TOOLS-2845'>TOOLS-2845</a>] -         Release Database Tools 100.4.0
+</li>
+</ul>
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
+<h3>        Bug
+</h3>
+<ul>
+<li>[<a href='https://jira.mongodb.org/browse/TOOLS-2041'>TOOLS-2041</a>] -         Mongorestore should handle duplicate key errors during oplog replay
+</li>
+<li>[<a href='https://jira.mongodb.org/browse/TOOLS-2833'>TOOLS-2833</a>] -         creating and index with partialFilterExpression during oplogReplay fails
```

Comment:
Small nit, I know the ticket title is incorrect but can we change this to "Creating an"?

Reply:
Sure!